macosx/InfoOptionsViewController.mm

```diff
@@ -295,7 +296,7 @@ - (void)updateOptions
             checkRatio = kInvalidValue;
         }
 
-        if (!multipleRatioLimits && ratioLimit != torrent.ratioLimit)
+        if (!multipleRatioLimits && isRatioEqual(ratioLimit, torrent.ratioLimit))
```

Comment:
Should be `!isRatioEqual` instead of `isRatioEqual`.

@ckerr you merged before the weekend review. -_-